d2graph/d2graph.go

```diff
@@ -805,7 +805,9 @@ func (obj *Object) GetDefaultSize(mtexts []*d2target.MText, ruler *textmeasure.R
 
 		// The rows get padded a little due to header font being larger than row font
 		dims.Height = labelDims.Height * (len(obj.SQLTable.Columns) + 1)
```

Comment:
Probably not to be fixed here as it is more of a compiler error than a rendering one, but `column\nname\nrow` will create line breaks in column names.
Of course this isn't valid in SQL, so we should handle it properly in the compiler, but it breaks the line heights